app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+class CommandError(Exception):
+    pass
+
+
+def copy_file(command: str) -> None:
+    command = command.split()
+    if len(command) != 3:
+        raise CommandError("Unsupported format of command")
+
+    cmd, file_name, new_file_name = command
+    if cmd == "cp":
+        if file_name != new_file_name:
```

Comment:
You can combine the two if conditions into a single line using the logical operator

Reply:
Oleg just said that I need to track both conditions and raise different exceptions